docs/trex_debugging.md

```diff
@@ -82,3 +82,6 @@ Due to changes in the start-up sequence, you can no longer enable remote debuggi
 #### Refreshed Page Fails
 Unfortunately, due to a bug with our embedded browser, refreshing a loaded page doesn't properly setup the communication between Tableau and the extension. This means that triggering a page reload will cause your extension to stop working. It's important to note that the **Reload** menu option actually tears down and re-creates the browser control, which means you'll need to establish a new debugging sessions whenever you click **Reload**. For more information, see [What Happens When You Reload an Extension]({{site.baseurl}}/docs/trex_reload.html).
 
+#### Extensions and Internet Explorer 11
+The Extensions API library uses modern JavaScript constructs, such as promises, arrow functions, typed arrays, and iterators. The library was built for ECMAScript 5 compatibility, and provides the appropriate polyfills. However, if you use ECMAScript 6 language constructs, your code will have syntax errors in Internet Explorer 11 (IE11). If you want your extension to run in IE11, you should avoid using ECMAScript 6 features and language constructs (see http://es6-features.org/) or have the appropriate polyfills and tools to convert to ECMAScript 5. If you are using Typescript, for example, TypeScript allows you to use ECMAScript 6 language constructs, but target ECMAScript 5 compatibility.
```

Comment:
Didn't Lee think this was too much information? I don't know what he recommended instead.

Reply:
@LGraber to see if he has any comments about what to leave out.  I sent this out for review a month ago but didn't hear back then.